manifests/00-ingress-credentials-request.yaml

```diff
@@ -37,4 +37,21 @@ spec:
     kind: AzureProviderSpec
     roleBindings:
     - role: passthrough
-      scope: resourcegroup
\ No newline at end of file
+      scope: resourcegroup
+---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: gcp-openshift-ingress
+  namespace: openshift-cloud-credential-operator
+spec:
+  secretRef:
+    name: cloud-credentials
+    namespace: openshift-ingress-operator
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: GCPProviderSpec
+    predefinedRoles:
+    - roles/dns.admin
```

Comment:
This is subject to change as we move forward — it was the narrowest usable prefab role I could find.